.travis.yml

```diff
@@ -36,8 +36,8 @@ env:
     - WELD_VERSION=2.0.3.Final
     - WELD_VERSION=2.0.2.Final
     - WELD_VERSION=2.0.1.Final
-    - WELD_VERSION=1.1.16.Final
-    - WELD_VERSION=1.1.15.Final
+  #  - WELD_VERSION=1.1.16.Final
+  #  - WELD_VERSION=1.1.15.Final
```

Comment:
@sventorben If we're getting rid of Weld 1.x support, these should just go. And so should the file Weld11TestUrlDeployment.java. Thanks!